package.json

```diff
@@ -20,19 +21,24 @@
     "node": "18.16.0"
   },
   "devDependencies": {
-    "@babel/core": "7.20.12",
-    "@babel/eslint-parser": "7.19.1",
```

Comment:
Neither of these dependencies are needed in the repository root. `@babel/eslint-parser` isn't needed at all (we use the typescript parser anyway).